### PR TITLE
WELD-2631 Destroy session context that can be left over from previous…

### PR DESCRIPTION
**Pull request by manovotn**
… timeouted session that ended exceptionally.

### CODE REVIEW

File: modules/web/src/main/java/org/jboss/weld/module/web/logging/ServletLogger.java [hunk 1]
```diff
@@ -97,4 +97,8 @@
     @Message(id = 718, value = "No EEModuleDescriptor defined for bean archive with ID: {0}. @Initialized and @Destroyed events for ApplicationScoped may be fired twice.", format = Format.MESSAGE_FORMAT)
     void noEeModuleDescriptor(Object beanArchiveId);
 
+    @LogMessage(level = Level.DEBUG)
+    @Message(id = 719, value = "When attempting to activate session context, an already active HttpSessionDestructionContext was detected. This is likely a leftover from previous sessions that ended exceptionally; this context will be terminated. ", format = Format.MESSAGE_FORMAT)
```
Comment (manovotn) [inline modules/web/src/main/java/org/jboss/weld/module/web/logging/ServletLogger.java:101]: @mkouba if you have an idea for better log msg, I am all ears

**Comment by manovotn** [comment]
FTR I have ran this through WFLY TS as well and saw no regressions.

File: modules/web/src/main/java/org/jboss/weld/module/web/logging/ServletLogger.java (hunk 1 above)
Comment (mkouba) [reply modules/web/src/main/java/org/jboss/weld/module/web/logging/ServletLogger.java:101]: Looks good but it's a bit too long. Maybe just "An active HttpSessionDestructionContext detected. This is likely a leftover from previous sessions that ended exceptionally; this context will be terminated."?

Comment (manovotn) [reply modules/web/src/main/java/org/jboss/weld/module/web/logging/ServletLogger.java:101]: +1, will adjust that

**Pull request merged**